express-sendfile/index.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const app = express();
+
+app.use((req, res, next) => {
+  if (req.originalUrl === '/index.html') {
+    res.sendFile('/home/dev/lfz/c0720-code-solutions/express-sendfile/index.html');
+  } else if (req.originalUrl === '/styles.css') {
+    res.sendFile('/home/dev/lfz/c0720-code-solutions/express-sendfile/styles.css');
+  } else if (req.originalUrl === '/main.js') {
+    res.sendFile('/home/dev/lfz/c0720-code-solutions/express-sendfile/main.js');
```

Comment:
this works, but it's not good to hard-code the absolute path, use the options object mentioned in step 7